app/renderer/components/navigation/navigationBar.js

```diff
@@ -115,6 +125,77 @@ class NavigationBar extends React.Component {
       this.props.bookmarked
   }
 
+  // BEM Level: navigationBar__navigationButtonContainer
+  get stopButton () {
+    return <button className={cx({
+      normalizeButton: true,
+      [css(styles.navigationButton, styles.navigationButton_stopButton)]: true
+    })}
+      data-l10n-id='stopButton'
+      onClick={this.onStop}
+    />
+  }
+
+  // BEM Level: navigationBar__navigationButtonContainer
+  get reloadButton () {
+    return <LongPressButton className={cx({
+      normalizeButton: true,
+      [css(styles.navigationButton, styles.navigationButton_reloadButton)]: true
+    })}
+      l10nId='reloadButton'
+      testId='reloadButton'
+      onClick={this.onReload}
+      onLongPress={this.onReloadLongPress}
+    />
+  }
+
+  // BEM Level: navigationBar__navigationButtonContainer
+  get homeButton () {
+    return <button className={cx({
+      normalizeButton: true,
+      [css(styles.navigationButton, styles.navigationButton_homeButton)]: true
+    })}
+      data-test-id='homeButton'
+      data-l10n-id='homeButton'
+      onClick={this.onHome}
+    />
+  }
+
+  // BEM Level: navigationBar
+  get bookmarkButtonContainer () {
+    return <span className={css(
+      commonStyles.navigationBar__buttonContainer,
+      commonStyles.navigationBar__buttonContainer_outsideOfURLbar,
+      styles.navigationBar__buttonContainer_bookmarkButtonContainer
+    )}>
+      <button className={cx({
+        normalizeButton: true,
+        withHomeButton: getSetting(settings.SHOW_HOME_BUTTON),
+        [css(styles.navigationBar__buttonContainer_bookmarkButtonContainer__bookmarkButton, this.bookmarked && styles.navigationBar__buttonContainer_bookmarkButtonContainer__bookmarkButton_removeBookmarkButton)]: true
+      })}
+        data-l10n-id={this.bookmarked ? 'removeBookmarkButton' : 'addBookmarkButton'}
+        data-test-id={this.bookmarked ? 'bookmarked' : 'notBookmarked'}
+        onClick={this.onToggleBookmark}
+      />
+    </span>
+  }
+
+  // BEM Level: navigationBar
+  get publisherToggleContainer () {
+    return <span className={css(
+      commonStyles.navigationBar__buttonContainer,
+      commonStyles.navigationBar__buttonContainer_outsideOfURLbar,
+      styles.navigationBar__buttonContainer_publisherToggleContainer
```

Comment:
Creating a custom component to remove commonStyles would be nice, such as:
````
<NavigationButtonContainer
  outURLbar
  className={css(
    styles.navigationBar__buttonContainer_publisherToggleContainer
  )}>
    ....
</NavigationButtonContainer>
````

@cezaraugusto what do you think? I thought it would be the similar/same approach as you took to create `BrowserButton` component.